sdc/crypto/test/test_jwe_helper.py

```diff
@@ -0,0 +1,173 @@
+import os
```

Comment:
I think there are a number things we can do to tidy up these tests and make them clearer, but happy for them to be done in another pull request.